appveyor.yml

```diff
@@ -179,3 +182,40 @@ on_finish:
   - 7z a %ARTIFACT% %FULL_GROONGA_INSTALL_FOLDER%
   - ps: Push-AppveyorArtifact $Env:ARTIFACT
   - cd ..
+
+# MinGW specific build
+for:
+-
+  matrix:
+    only:
+      - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+        MINGW_CHOST: mingw-w64-x86_64
+        MINGW_PREFIX: /mingw64
+  install:
+    - set PATH=C:\msys64\mingw64\bin;c:\msys64\usr\bin;%PATH%
+    - choco install -y 7zip.commandline
+    - pacman -Syu --noconfirm
+    - pacman -S --noconfirm --needed
+        autoconf-archive
+        mingw-w64-x86_64-gettext
+        mingw-w64-x86_64-python3-colorama
+        mingw-w64-x86_64-python3-sphinx
+        mingw-w64-x86_64-ruby
+    - ps: |
+        $Env:GROONGA_VERSION = (Get-Content base_version)
+  build_script:
+    - bash -c "./autogen.sh"
+    - bash -c "./configure --host=x86_64-w64-mingw32 --target=x86_64-w64-mingw32 --build=x86_64-w64-mingw32 --enable-document --with-ruby=ruby --enable-mruby"
+    - bash -c "make dist"
+    - cd %APPVEYOR_BUILD_FOLDER%\..
+    - mkdir build
+    - cd build
+    - cp %APPVEYOR_BUILD_FOLDER%\appveyor\PKGBUILD .
+    - cp %APPVEYOR_BUILD_FOLDER%\groonga-%GROONGA_VERSION%.tar.gz .
+    - bash -c "GROONGA_VERSION=%GROONGA_VERSION% MINGW_INSTALLS=mingw64 makepkg-mingw -sLf --noconfirm --needed"
```

Comment:
Why do we need to use `makepkg-mingw`?

Reply:
Because build error issue is found on PKGBUILD environment. If `makepkg-mingw` doesn't affect it, there is no need to use `makepkg-mingw`.

Reply:
I've tried without makepkg-mingw. It needs to be fixed additional linkage error issue.
https://ci.appveyor.com/project/kenhys/groonga/builds/26000711/job/l79vkjnpkrtsmgrm

Reply:
It seems that using `makepkg-mingw` is better instead of directly using ./configure ... because it absorbs build related bothersome things.